src/openrct2-ui/windows/MazeConstruction.cpp

```diff
@@ -363,16 +363,19 @@ static void window_maze_construction_entrance_tooldown(ScreenCoordsXY screenCoor
     gMapSelectFlags &= ~MAP_SELECT_FLAG_ENABLE_ARROW;
 
     int32_t direction = 0;
+    CoordsXY entranceOrExitCoords;
     ride_get_entrance_or_exit_position_from_screen_position(
-        screenCoords.x, screenCoords.y, &screenCoords.x, &screenCoords.y, &direction);
+        screenCoords.x, screenCoords.y, &entranceOrExitCoords.x, &entranceOrExitCoords.y, &direction);
+    if (entranceOrExitCoords.x == 0x8000)
```

Comment:
Use the location_null macro. Could also fix ride get entrance or exit position to use the macro at the same time

Reply:
I am using the macro already, note that I did two commits, you probably saw an outdated version

Reply:
How odd. Must have been some cache issue